src/librustc_codegen_ssa/Cargo.toml

```diff
@@ -13,7 +13,6 @@ test = false
 bitflags = "1.0.4"
 cc = "1.0.1"
 num_cpus = "1.0"
-rustc-demangle = "0.1.15"
```

Comment:
is `rustc-demangle` not intentionally for some reason?

Reply:
Looks like it was never used, cc @eddyb https://github.com/rust-lang/rust/commit/b9e5cf99a9acd9c29d02c2f27c0eb223fd0a92be

Reply:
Copy-pasted but the only use is still in `rustc_codegen_llvm`.